pdr_backend/analytics/predictoor_stats.py

```diff
@@ -229,9 +229,11 @@ def get_mean_slots_slots_df(slots_df: pl.DataFrame) -> pl.DataFrame:
     slots_daily_df = (
         slots_df.group_by(["pair_timeframe", "datetime"])
         .map_groups(
-            lambda df: get_mean_slots_slots_df(df.sample(5))
-            if len(df) > 5
-            else get_mean_slots_slots_df(df)
+            lambda df: (
```

Comment:
Why is this here? Better to be in a separate change. Avoids confusing Mustafa / Roberto / others.